{{cookiecutter.profile_name}}/lsf_status.py

```diff
@@ -109,5 +109,5 @@ def get_status(self) -> str:
 if __name__ == "__main__":
     jobid = int(sys.argv[1])
     outlog = sys.argv[2]
-    lsf_status_checker = LSF_Status_Checker(jobid, outlog)
+    lsf_status_checker = LSF_Status_Checker(jobid, outlog, 0.001, 1)
```

Comment:
Could you please name the variables? i.e. `LSF_Status_Checker(jobid, outlog, wait=0.001, retry=1)`

Maybe while you're at it could you make those internal member variable names lowercase?

Reply:
nm I will do it